joomla/joomla-cms
# Add missing 'Show Subcategories Text' param to Contacts

Pull Request for Issue # .

### Summary of Changes
For Contacts component:
/administrator/components/com_contact/config.xml - add missing ```show_category_heading_title_text``` parameter.
/components/com_contact/views/categories/tmpl/default.xml - add missing ```show_category_heading_title_text``` parameter, code style.
/components/com_contact/views/category/tmpl/default.xml - add missing ```show_category_heading_title_text``` parameter, code style.

### Testing Instructions
Use a fresh Joomla, apply the patch.
Create a tree of some contact categories.
Create contacts with default options and place they to some contact categories.
Create menu items of the types Contacts » List All Contact Categories and Contacts » List Contacts in a Category for the root contact category.
Check the contact categories in the frontend. All should be fine.
Play with the option 'Show Subcategories Text' in the Global Configuration » Contacts, in the Category tab of the menu items you created.

NB!: When testing the  menu item of the type ```Contacts » List All Contact Categories```, the menu item of the type ```Contacts » List Contacts in a Category```  must be unpublished.

### Expected result
When the option 'Show Subcategories Text' is Off in backend, the label "Subcategories" must not be shown in frontend.

### Actual result
The option 'Show Subcategories Text' is missing in backend, the label "Subcategories" is shown always in frontend.

### Documentation Changes Required
